src/main/java/com/autotune/common/annotations/json/AutotuneJSONExclusionStrategy.java

```diff
@@ -0,0 +1,18 @@
+package com.autotune.common.annotations.json;
```

Comment:
LICENSE header missing

Reply:
Thanks for the review @dinogun . Added the LICENSE Header